opencga-analysis/src/main/java/org/opencb/opencga/analysis/alignment/AlignmentCoverageAnalysis.java

```diff
@@ -140,7 +141,16 @@ protected void run() throws Exception {
 
             // And finally, link the BW file is necessary
```

Comment:
Instead of manually "moving and linking" the file, there is a method `OpenCgaTool#moveFile` that you could use, which moves, links and associates the file with the job